app/domain/export/pdf/passes/membership/footer.rb

```diff
@@ -9,14 +9,136 @@ class Export::Pdf::Passes::Membership
   class Footer < Export::Pdf::Section
 
     def render
-      pdf.move_down(50)
-      image(sponsors_image_path, align: :center)
+      text_box title,
+               at: [61.409, 155.561],
+               align: :left,
+               size: 10,
+               style: :bold
+
+      draw_back
+      draw_dashed_box
     end
 
     private
 
-    def sponsors_image_path
-      image_path('sponsors_membership_pass.png')
+    def draw_back
+      pdf.image(reciprocity_image_path, position: :left, at: [496.714, 190.213],
+                                        width: 39.885)
+
+      draw_sponsor_code
+
+      right_side_left_start = 309.425
+
+      text_box 'REGA 1414                    SOS Europe 112',
+               at: [right_side_left_start,
+                    188.1799],
+               style: :bold,
+               size: 8
+
+      write_in_boxes(right_side_left_start)
+
+      sac_cas_link
+    end
+
+    def sac_cas_link
+      fill_color('ED1C24') do
+        text_box 'www.sac-cas.ch',
+                 at: [489.973,
+                      153.408],
+                 size: 7
+      end
+    end
+
+    def draw_sponsor_code
+      sponsor_qr_code_width = 55
+      pdf.image(sponsor_qr_code, position: :left, at: [499, 88.126],
+                                 width: sponsor_qr_code_width)
+      text_box t('sac_partner'),
+               at: [499,
+                    87],
+               align: :center,
+               size: 6,
+               width: sponsor_qr_code_width
+    end
+
+    def write_in_boxes(right_side_left_start)
+      text_box_width = 170.017
+      text_box_height = 28.063
+      gray_boxes_y_position = [175.307, 144.307]
+
+      fill_color('EFEFEF') do
+        gray_boxes_y_position.each do |y_position|
+          pdf.fill_rectangle [right_side_left_start, y_position], text_box_width,
+                             text_box_height
+        end
+      end
+
+      fill_color('231f20') do
+        ["Notfallnummer / N° d'urgence / No. di emergenza",
```

Comment:
Fände ich persönlich besser in den locales, zb  unter `global.emergency_contact_number` und dann mit `I18n.with_locale(..) { I18n.t('global.emergency_contact_number') }` auslesen